quickwit/quickwit-doc-mapper/src/doc_mapper.rs

```diff
@@ -236,7 +249,7 @@ mod tests {
             sort_by_field: None,
             aggregation_request: None,
         };
-        let query = doc_mapper.query(schema, &search_request).unwrap();
+        let (query, _) = doc_mapper.query(schema, &search_request).unwrap();
```

Comment:
do we unit test the Warmup structure somewhere?
This file seems like a good place to do that.

Reply:
latest commit add a simple test case [in `query_builder.rs`](https://github.com/quickwit-oss/quickwit/pull/2284/commits/8e248ca698c8fdfb55a8e4f0026fed588b8dd98b#diff-c7130467f6b55c09737ded5297aa3d1b083d5e4913fc97874d2bc95af18b8baaR502). Maybe `doc_mapper.rs` is a better place